src/main.rs

```diff
@@ -135,7 +136,24 @@ fn main() -> Result<()> {
 
     let query_string = opts.read_query_to_string().transpose()?;
 
-    let input = BytesContainer::new(BytesContainerType::Input, opts.codec, buffer)?;
+    let engine = Engine::new(
+        Config::new()
+            .wasm_multi_memory(true)
+            .wasm_threads(false)
+            .consume_fuel(true)
+            .epoch_interruption(true),
```

Comment:
To ensure this configuration is consistent within this program, I wonder if we actually want to just default to this when creating the engine and not make it configurable in `new`. If we do have a use-case for configuring the engine differently in different spots in the future, we can change it then.